ddl/ddl_api.go

```diff
@@ -1023,6 +1023,19 @@ func (d *ddl) CreateTable(ctx sessionctx.Context, s *ast.CreateTableStmt) (err e
 	return errors.Trace(err)
 }
 
+func (d *ddl) RestoreTable(ctx sessionctx.Context, tbInfo *model.TableInfo, schemaID, autoID, dropJobID int64, snapshotTS uint64) (err error) {
+	tbInfo.State = model.StateNone
+	job := &model.Job{
+		SchemaID:   schemaID,
+		TableID:    tbInfo.ID,
+		Type:       model.ActionRestoreTable,
+		BinlogInfo: &model.HistoryInfo{},
+		Args:       []interface{}{tbInfo, autoID, dropJobID, snapshotTS, restoreTableCheckFlagNone},
```

Comment:
Why add the `restoreTableCheckFlagNone` parameter here?

Reply:
`restoreTableCheckFlagNone` is a flag, use to check gc enable status in `onRestoreTable` step 1.

Reply:
`restoreTableCheckFlagNone` is a flag, use to check gc enable status in `onRestoreTable` step 1.

Reply:
It feels like using parameters to occupy position.